### ISSUE TITLE
Accessibility for colorblind users

### ISSUE THREAD

**Issue by zertrin** [gitahead/gitahead#392]
The colors used in the UI should be reviewed a bit for accessibility for color-blind users.

The prevalence of color-blindness is much higher that most would think. (See table [here](https://www.colour-blindness.com/general/prevalence/), about 1 in 10 to 1 in 12 men have some sort of color-blindness). Note that this is not a black/white (pun not intended) situation, rather a spectrum, as some people have mild color-blindness while others are more affected than others.

For example I know I have slight color-blindness for red-green, but in the majority of cases I can still distinguish them. Only in a few situations it becomes hard to distinguish them apart.

And GitAhead which I have just started using uses shades of red and green for the diff that I find harder to distinguish compared to other Git GUI tools. (I have just migrated from GitKraken, where I didn't have such an issue).

On the screenshot below for example, I find it hard to tell spontaneously whether the added lines 502-503 are red or green.

On lines 513-516, I perceive some color difference, but need to know the context to realize that the upper is red and bottom is green.

The main issue here in my opinion is that the luminance values of the red and green colors used are too close from each other. In other tools the luminance of the red versus the one of the green is more pronounced, such that it is easier to learn the difference and recognize it immediately.

For example on the image below, where the white-space changes are highlighted at the beginning of lines 520/522, the red and green colors are much easier for me to distinguish than the rest of the line.

![image](https://user-images.githubusercontent.com/1447163/76180659-838c9180-61f9-11ea-9507-a7fa2e627f00.png)

**Comment by The-EDev** [comment]
you can use your own theme, or download a pre made theme that fixes this problem, using Tools -> Options -> Window and change theme to whatever you want it to be, you would need to use hex colors but that shouldn't be a problem

**Comment by zertrin** [comment]
Thanks for the info, I wasn't aware of this possibility 👍. This can be a temporary workaround.

However I still think that the defaults should be optimized a bit (do we expect 7-10% of male users to have to tweak the style settings themselves?).

**Comment by The-EDev** [comment]
If you'd like, this is the file for the dark theme: https://github.com/gitahead/gitahead/blob/master/conf/themes/Dark.lua

it's basically the same layout as creating a custom theme, you can change the colors there and create a pull request, which would be awesome

the lines for the diffs start at 126, specifically addition, deletion, word_addition, word_deletion

**Comment by zertrin** [comment]
Yes I found by myself from the instructions in your previous comment. I have tweaked the colors to something I can distinguish much easier:

![image](https://user-images.githubusercontent.com/1447163/76305645-7ad2b300-6300-11ea-8cb9-64534c1e9fbc.png)

I changed addition, deletion, word_addition, word_deletion indeed:

```lua
-- diff view colors
theme['diff']      = {
  addition         = '#006000', -- added lines
  deletion         = '#500000', -- deleted lines
  plus             = '#207A00', -- plus icon
  minus            = '#BC0009', -- minus icon
  ours             = '#000060', -- ours conflict lines
  theirs           = '#600060', -- theirs conflict lines
  word_addition    = '#238400', -- added words
  word_deletion    = '#800000', -- deleted words
  note             = '#E1E5F2', -- note squiggle
  warning          = '#E8C080', -- warning background
  error            = '#7E494B'  -- error background
}
```

⚠️ Disclaimer: my color choice is NOT meant to be suitable for a new default. I am only slightly color-blind, and my choice of colors might not be suited for other people. Just provided here as an example for other people.
